docs/release-notes/7.17.asciidoc

```diff
@@ -2,14 +2,25 @@
 == 7.17
 
 [discrete]
-[[release-notes-7.17.1]]
-=== 7.17.1
+[[release-notes-7.17.2]]
+=== 7.17.2
 
 [discrete]
-[[bug-fixes-7.17.1]]
+[[bug-fixes-7.17.2]]
 ==== Bug fixes and enhancements
+* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.
+* The bug that prevented Endpoint Security from being run on Windows 8.1 and Server 2012 R2 has been fixed. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc (https://github.com/elastic/endpoint/issues/15[#15]).
```

Comment:
I'm sure it's typographically awful, but there is a period at the end of the new signer name

```suggestion
* The bug that prevented Endpoint Security from being run on Windows 8.1 and Server 2012 R2 has been fixed. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).
```

Reply:
Good catch, thanks! I accidentally deleted the period when adding the PR link.